base/uk.ac.stfc.isis.ibex.ui.blocks/src/uk/ac/stfc/isis/ibex/ui/blocks/groups/BlocksMenu.java

```diff
@@ -113,12 +120,27 @@ public void run() {
      *
      * @param displayBlock the selected block
      */
-    public BlocksMenu(DisplayBlock displayBlock) {
+    public BlocksMenu(DisplayBlock displayBlock, IHandlerService handlerService) {
 		this.block = displayBlock;
 		
 		Configurations.getInstance().server().setCurrentConfig().addOnCanWriteChangeListener(readOnlyListener);
 
         add(new GroupMarker(BLOCK_MENU_GROUP));
+          
+        final IAction viewRunControlSettingsAction = new Action(VIEW_RUN_CONTROL_SETTINGS) {
```

Comment:
Do you think it's worth adding a brief comment here just to describe the below block?